filament/backend/include/backend/Program.h

```diff
@@ -90,6 +90,10 @@ class Program {
     // the program everything it needs to know about the uniforms at a given binding
     Program& uniforms(uint32_t index, UniformInfo const& uniforms) noexcept;
 
+    // Note: This is only needed for GLES2.0.
+    Program& attributes(
+            utils::FixedCapacityVector<std::pair<utils::CString, uint8_t>> attributes) noexcept;
```

Comment:
Make `attributes` a `const&` then simply assign to `mAttributes` instead of moving? This is consistent with the other methods of `Program`.